src/objects/Error__c.object

```diff
@@ -144,6 +144,29 @@
         <trackTrending>false</trackTrending>
         <type>LongTextArea</type>
         <visibleLines>10</visibleLines>
+    </fields>
+     <fields>
+        <fullName>Record_Id__c</fullName>
+        <description>The Salesforce Id of the record associated with this error.</description>
+        <externalId>true</externalId>
+        <inlineHelpText>The Salesforce Id of the record associated with this error.</inlineHelpText>
+        <label>Record Id</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Retry_Pending__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>The record associated with this error will be reprocessed the next time the process the error is associated with executes.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The record associated with this error will be reprocessed the next time the process the error is associated with executes.</inlineHelpText>
```

Comment:
Should this state somewhere that it's for Gem reprocessing only (or something that indicates that not every error will support Retrying this way)?

Reply:
I think it should absolutely say something, but I am not certain we should say GEM/Abacus/Payment Allocations only...

Reply:
Agreed. We can let docs come up something that makes sense.

Reply:
^^^ @markross-sfdo